src/sas/system/log.ini

```diff
@@ -41,7 +41,7 @@ args=(os.path.join(os.path.expanduser("~"),'sasview.log'),"a")
 # Loggers
 
 [loggers]
-keys=root,saspr,sasgui,sascalc,sasmodels
+keys=root,saspr,sasgui,sascalc,sasmodels,h5py
```

Comment:
h5py is not "ours", so it has no place in these keys.
3rd party loggers should probably be all set by default to at least DEBUG if not WARNING, with those that need detailed logging whitelisted.
https://github.com/SasView/sasview/pull/2275#discussion_r1005979992

Reply:
@rozyczko If you know a better way to do it, please do. I'm definitely not sure the solution I proposed is good.

Reply:
I asked everyone downstairs if they knew the right way of disabling these messages, and nobody did.

Reply:
Changing lines 36 and 47 from DEBUG to WARNING didn't do it?

Reply:
That works, is generally better.